projects/plugins/jetpack/modules/gravatar-hovercards.php

```diff
@@ -274,6 +274,16 @@ function grofiles_attach_cards() {
 			$my_hash = '';
 		}
 		wp_localize_script( 'wpgroho', 'WPGroHo', compact( 'my_hash' ) );
+		wp_localize_script(
+			'grofiles-cards',
+			'GravatarHovercardsi18n',
+			array(
+				'Edit your profile' => __( 'Edit your profile', 'jetpack' ),
+				'View profile'      => __( 'View profile', 'jetpack' ),
+				'Sorry, we are unable to load this Gravatar profile.' => __( 'Sorry, we are unable to load this Gravatar profile.', 'jetpack' ),
+				'Sorry, we are unable to load this Gravatar profile. Please check your internet connection.' => __( 'Sorry, we are unable to load this Gravatar profile. Please check your internet connection.', 'jetpack' ),
```

Comment:
Shouldn't the keys be JavaScript variables instead?

Reply:
No, it's a map of English text to translation.